dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeHistoricalFileHelper.java

```diff
@@ -0,0 +1,152 @@
+package com.ibm.ws.security.acme.internal;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.security.cert.X509Certificate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.StringTokenizer;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.security.acme.AcmeCaException;
+import com.ibm.ws.security.acme.AcmeCertificate;
+import com.ibm.wsspi.kernel.service.location.WsLocationAdmin;
+
+public class AcmeHistoricalFileHelper {
+	private static final TraceComponent tc = Tr.register(AcmeHistoricalFileHelper.class);
+	private String spaceDelim = "                  ";
+	
+	/**
+	 * This method determines if the directoryURI has changed by checking
+	 * the acme file.
+	 * @return True if the directoryURI has changed
+	 * @throws AcmeCaException
```

Comment:
Does this really throw an AcmeCaException?

Reply:
nope